app-text/cmigemo/cmigemo-1.3c-r3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit flag-o-matic toolchain-funcs
+
+MY_P="${P}-MIT"
+DESCRIPTION="C/Migemo -- Migemo library implementation in C"
```

Comment:
We don't duplicate the package name in description.

Reply:
fixed